### PR TITLE
Doc/update readme

### PR DESCRIPTION
**Pull request by PaigeCD**
# Update README

## Description

Make requested changes to README.md

### Type of Change

- [ ] Feature
- [x] Bug fix
- [x] Documentation

## Contributors

- @PaigeCD 
- @favourojo 

## Reminder

All GitHub Actions should be in a passing state before any pull request is merged.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,6 +16,50 @@ spelling with [cspell](https://cspell.org/).
...
+`Wrote file to: C:\Users\<YOUR PATH>\config\gatorgrader.yml`
+
+This command will auto-generate a default configuration file for GatorGradle
+named `gatorgrader.yml` located in the `config` folder.
+
+Additionally, you can run the `gatorconfig --help` for more
+information about the configuration. This command will list the variables
+in the file as well as the defaults it outputs.
+
+## Contributing
```
Comment (ColemanKobe) [inline README.md:52]: If you want to be real fancy, you could create a CONTRIBUTING.md file as well.

**Review by ColemanKobe — CHANGES_REQUESTED** [review]
This is good so far, but I think the README could be a bit more clear on this point before merging: `Where users can get help with the project.` Probably with some specific aspects of the tool itself, such as consulting Danny for the GUI section, consulting Wesley for the CLI section, etc.

File: README.md (hunk 1 above)
Comment (PaigeCD) [reply README.md:52]: Get help with what exactly? Do you mean that the explanation for running the program needs more details?

Comment (ColemanKobe) [reply README.md:52]: For example, if someone wants to contribute to the GUI, they can consult Danny for further details on the overall design of it.

Comment (PaigeCD) [reply README.md:52]: Oh, I understand now. Would you suggest that the author's section specifies this? Or maybe a note under the Contributing heading?

Comment (ColemanKobe) [reply README.md:52]: The author's section should be good enough. It's concise.

**Pull request merged**